src/puppetlabs/puppetdb/reports.clj

```diff
@@ -154,29 +140,20 @@
 ;;; Reports Query -> Wire format conversions
 
 (pls/defn-validated resource-events-query->wire-v5 :- [resource-event-v5-wireformat-schema]
-  [events :- resource-events-expanded-query-schema]
-  (sort-by
-   #(mapv % [:timestamp :resource_type :resource_title :property])
-   (map
-    #(dissoc % :report :certname :containing_class :configuration_version
-             :run_start_time :run_end_time :report_receive_time :environment)
-    (:data events))))
-
-(pls/defn-validated logs-query->wire-v5 :- [log-wireformat-schema]
-  [logs :- logs-expanded-query-schema]
-  (:data logs))
-
-(pls/defn-validated metrics-query->wire-v5 :- [metric-wireformat-schema]
-  [metrics :- metrics-expanded-query-schema]
-  (:data metrics))
+  [resource-events :- resource-events-expanded-query-schema]
+  (->> resource-events
+       :data
```

Comment:
Although this clearly works, the code implies to me that `resource-events` is itself a sequence. Leading off with `(->> (resource-events :data) ...)` and going straight into map aligns more clearly with the "`->>` is used for sequences" heuristic.